ruby/README.md

```diff
@@ -0,0 +1,64 @@
+# HubSpot-ruby CRM-objects sample app
+
+### Requirements
+
+1. ruby 2.6.3
+2. [Configured](https://github.com/HubSpot/sample-apps-manage-crm-objects/blob/main/README.md#how-to-run-locally) .env file
+
+### Running
+
+1. Install dependencies
+
+```
+bundle install
+```
+
+2. Commands
+
+Show all commands (get help)
+
+```
+ruby cli.rb -h
```

Comment:
This is not something required, but to keep structure consistent through the apps, I'd suggest running similar to other languages:

```
./bin/cli.ruby
```

Reply:
Could be done in next PRs though